src/main/java/reposense/authorship/FileInfoExtractor.java

```diff
@@ -123,14 +132,16 @@
     }
 
     /**
-     * Returns a {@code Set} of non-binary files for the repo {@code repoConfig}.
+     * Returns a {@code Set} of non-binary files for the repo {@code repoConfig}
+     * if {@code isBinaryFiles} is set to `false`.
+     * Otherwise, returns a {@code Set} of binary files for the repo {@code repoConfig}
      */
-    public static Set<Path> getNonBinaryFilesList(RepoConfiguration repoConfig) {
+    public static Set<Path> getFilesList(RepoConfiguration repoConfig, boolean isBinaryFiles) {
         List<String> modifiedFileList = GitDiff.getModifiedFilesList(Paths.get(repoConfig.getRepoRoot()));
 
-        // Gets rid of binary files and files with invalid directory name.
+        // Gets rid of files with invalid directory name and filters by whether file is binary.
         return modifiedFileList.stream()
-                .filter(file -> !file.startsWith(BINARY_FILE_LINE_DIFF_RESULT))
+                .filter(file -> isBinaryFiles == file.startsWith(BINARY_FILE_LINE_DIFF_RESULT))
                 .map(rawNonBinaryFile -> rawNonBinaryFile.split("\t")[2])
```

Comment:
how would this affect binary file?

Reply:
Thanks for pointing that out.

This is the output for non binary file and binary file respectively.
```
3	0	src/test/resources/repoconfig_merge_test/repo-config.csv
-	-	docs/diagrams/architecture.pptx
```
So both types of file need it to work.

Reply:
I will change the variable name to `file`